### ISSUE TITLE
[FEATURE REQUEST] Replace default Windows 11 clock

### ISSUE THREAD

**Issue by ClaudiuSchuster** [martinet101/ElevenClock#98]
Hi, 

currently if you use "TaskbarSi=0" the main-screen clock is pretty fucked up, is it possible to replace the default clock also with elevenclock (would be also nice for adding seconds to default clock)?
![tray](https://user-images.githubusercontent.com/13591392/137182708-834ded78-3325-4808-99c4-0a011302249c.jpg)

**Comment by martinet101** [comment]
I don't think I can hide the default clock and set a space for ElevenClock to show, but anyway I'll give it a try.

P. D. Try rebooting windows, it may fix the clock

**Comment by Auncaughbove17** [comment]
is there a way to show the weekday on a seperate line, like the elevenclock setting, in windows 11?
That's why I looked into replacing it.

**Comment by martinet101** [comment]
Yes, but that would require a dynamic size (which I'm working on it, see the "centering the text in the clock" issue), and also to recreate mica effect, which I have not found any way to do, but Im still looking for it (bc the settings window would look fairly better with Mica)

**Comment by Auncaughbove17** [comment]
So all hope is not lost. Thanks for doing Microsoft's work and implementing features they are missing. :)
